### PR TITLE
Hw06 email

### PR DESCRIPTION
**Pull request by Oksana-Pelekhata**
Доброго дня!
Прохання перевірити.

### CODE REVIEW

File: controllers/auth-controller.js [hunk 1]
```diff
@@ -2,44 +2,90 @@ import bcrypt from "bcryptjs"
...
 
+const verify = async (req, res) => {
+    const { verificationToken } = req.params;
+    const user = await User.findOne({ verificationToken })
+    
+    if (!user) {
+        throw HttpError (404, "Email is not found")
+    }
+
+    await User.findByIdAndUpdate(user._id, { verify: true, verificationToken: "" })
```
Comment (ceREAL133) [inline controllers/auth-controller.js]: verificationToken should be null

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: controllers/auth-controller.js (hunk 1 above)
Comment (Oksana-Pelekhata) [reply controllers/auth-controller.js]: changed to null

Comment (ceREAL133) [reply controllers/auth-controller.js]: have you tested it? Will it work correctly?

Comment (Oksana-Pelekhata) [reply controllers/auth-controller.js]: It seems like it is working correctly.
I've attached screens to my homework.

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**